app/assets/scripts/components/new-atbd/index.js

```diff
@@ -0,0 +1,362 @@
+  font-weight: bold;
+  max-width: 32rem;
+`;
+
+const Feedback = styled.div`
+  text-align: center;
+`;
+
+const FeedbackLink = (props) => (
+  <a
+    href='#'
+    title='Open feedback form modal'
+    onClick={(e) => {
+      e.preventDefault();
+      document.dispatchEvent(new Event('show-feedback-modal'));
+    }}
+    {...props}
+  />
+);
+
+function NewAtbd() {
+  const [showMoreInfo, setShowMoreInfo] = React.useState(false);
+  const { user } = useUser();
+
+  return (
+    <App pageTitle='New ATBD'>
+      <PageContent>
+        <Header>
+          <h1>ATBD Creation Choices</h1>
+          {!user.isLogged && (
+            <Feedback>
+              <a href={getHostedAuthUiUrl('signup')}>Sign up</a> now to get
+              started!
+            </Feedback>
+          )}
+          <LearnMore>
+            Unsure whether the user interface or template is right for you?
+            Click{' '}
+            <Link
+              as='button'
+              to='#'
+              onClick={() => {
+                setShowMoreInfo(true);
+              }}
+            >
+              here
+            </Link>{' '}
+            to learn more about both.
+          </LearnMore>
+        </Header>
+        <SectionContainer>
+          <Section>
+            <h2>APT User Interface</h2>
+            <p>
+              Create and publish an ATBD from start to finish using the APT user
+              interface.
+            </p>
+            <div>
+              See <Link to='/user-guide'>APT user guide</Link>
+            </div>
+            <Video
+              src=''
+              title='APT - creating a document'
+              frameborder='0'
+              allowfullscreen
+            />
+          </Section>
+          <Section>
+            <h2>ATBD Templates</h2>
+            <p>
+              Create an ATBD using a standardized template. Upload the completed
+              document and basic metadata to APT. An account is required for
+              upload.
+            </p>
+            <div>
+              See <Link to='/'>ATBD template user guide</Link>
+              <em> (coming soon)</em>
+            </div>
+            <TemplateContainer>
+              <TemplateLink
+                href='https://docs.google.com/document/d/1T4q56qZrRN5L6MGXA1UJLMgDgS-Fde9Fo4R4bwVQDF8/edit?usp=sharing'
+                target='_blank'
+                rel='noopener'
+              >
+                <span>
+                  <FaFileAlt />
+                </span>
+                <div>Google Docs</div>
+              </TemplateLink>
+              <TemplateLink
+                href='https://github.com/NASA-IMPACT/nasa-apt/files/11421194/APT.ATBD.Template.docx'
+                target='_blank'
+                rel='noopener'
+              >
+                <span>
+                  <FaFileWord />
+                </span>
+                <div>Microsoft Word</div>
+              </TemplateLink>
+              <TemplateLink href='/'>
```

Comment:
@sunu also here. I'm not sure the best way for us to host these files. Thoughts? Or @batpad?